packages/plugin-lighthouse/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+## Setup
+
+Make sure to install dependencies:
+
+```sh
+npm install
+```
+
+### Chrome path
```

Comment:
Thank you for taking the time to account for this and document it 🙏 
It works for me now 😁 🎉

Reply:
Nice to here!!